php_mongo.c

```diff
@@ -259,7 +259,7 @@ PHP_MINIT_FUNCTION(mongo)
 	REGISTER_LONG_CONSTANT("MONGO_STREAMS", 1, CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("MONGO_SUPPORTS_STREAMS", 1, CONST_PERSISTENT);
 
-#if HAVE_OPENSSL_EXT
+#if HAVE_MONGO_OPENSSL
```

Comment:
typo alert. this needs to be ifdef

Reply:
I thought so earlier, but apparently not. This is no different than what we do with `#if HAVE_MONGO_SASL`, which is only `AC_DEFINE`-ed as `1` if SASL is detected (and never defined otherwise).

https://stackoverflow.com/a/5085425/162228 talks a bit about this.

Reply:
I think the sasl case is a bug too.
Mixing is-defined and is-postive-value is not a good idea.

Reply:
I'll change these to `#ifdef` if there are no objections.

Reply:
Just because someone else does doesn't make it right :)

There is no need to change this since it does work for now, I was merely attempting to point out an accidental typo. Did not realize it was intentional.

Reply:
Agreed that `#if` should definitely be `#ifdef`. I was only providing examples to demonstrate that it does currently work.